django/contrib/admin/static/admin/js/admin/DateTimeShortcuts.js

```diff
@@ -19,9 +19,8 @@
         timezoneWarningClass: 'timezonewarning', // class of the warning for timezone mismatch
         timezoneOffset: 0,
         init: function() {
-            var body = document.getElementsByTagName('body')[0];
-            var serverOffset = body.getAttribute('data-admin-utc-offset');
-            if (serverOffset !== undefined) {
```

Comment:
Should work too and seems more fitting than the extra method. I'll implement this in both checks and remove the serverOffset method.